### ISSUE TITLE
Are you HIT student？

### ISSUE THREAD

**Issue by lixryjz** [xerifg/Real-time-3DposeTracking-Unity3D-UnityChan#5]

**Comment by xerifg** [comment]
nope.lm NEU-------- 原始邮件 --------发件人： lixryjz ***@***.***>日期： 2022年11月14日周一 下午2:05收件人： xerifg/Real-time-3DposeTracking-Unity3D-UnityChan ***@***.***>抄送： Subscribed ***@***.***>主    题： [xerifg/Real-time-3DposeTracking-Unity3D-UnityChan] Are you HIT student？ (Issue #5)

**Comment by lixryjz** [comment]
what do you want to say???

**Comment by xerifg** [comment]
> what do you want to say???

sorry,there was some errors.im from NEU .hhhhh
